git_version/templatetags/git_tags.py

```diff
@@ -5,11 +5,15 @@
 
 @register.simple_tag
 def current_commit(path):
-    repo = Repo(path)
-    commit = repo.commit()
-    chash = commit.hexsha
-    repo.__del__()
-    return chash
+    try:
+        repo = Repo(path)
+        commit = repo.commit()
+        chash = commit.hexsha
+        repo.__del__()
+        return chash
+    except Exception:
```

Comment:
```suggestion
    except Exception as e:
```
Is there a specific reason not to print the actual exception message? Was that too verbose?